### PR TITLE
setup for sample apps

### PR DESCRIPTION
**Pull request by dannysheridan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# TODO Clients
+# Climatiq Clients
 
 This repository contains:
 
-- [Fern API Definition](./api/src/TODO.yml) of the TODO API
+- [Fern API Definition](./api/src/) of the Climatiq API
```
Comment (dsinghvi) [inline README.md:5]: should there be a link here to the published postman collection?

**Comment by dsinghvi** [comment]
have you used the postman collection to hit the api and confirmed it works? (if it does, then i can test with the generated sdks)

**Comment by dannysheridan** [comment]
Yes. I've used the Postman Collection to hit the example requests in their Docs. I've confirmed the sample responses.

File: README.md (hunk 1 above)
Comment (dannysheridan) [reply README.md:5]: Yes. Just checked in the Postman Collection and made the change to reference it here.

**Pull request merged**